scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -247,15 +241,27 @@
   cc/ComputeCluster
   (launch-tasks [this offers task-metadata-seq]
     (doseq [task-metadata task-metadata-seq]
-      (let [pod-namespace (get-namespace-from-task-metadata namespace-config task-metadata)]
+      ; Has the workload of launching tasks changed or has the cost of doing them changed?
+      ; Note we can't use timer/time! because it wraps the body in a Callable, which rebinds 'this' to another 'this'
+      ; causing breakage.
+      ; In addition, it capture controller/update-cook-expected-state and
+      ; screw up when we do a with-redefs on it in unit tests.
```

Comment:
I don't understand what this part means - can we clarify or remove?

Reply:
Reworded.